src/lib/sw_engine/tvgSwRaster.cpp

```diff
@@ -47,6 +47,18 @@ static inline uint32_t _ialpha(uint32_t c)
 }
 
 
+static inline uint32_t _abgrGrayValue(uint32_t c)
+{
+    return ((((c&0xff)*54) + (((c>>8)&0xff)*183) + (((c>>16)&0xff)*19)) + 0xff) >> 8; //0.2125*R + 0.7154*G + 0.0721*B
```

Comment:
0xff - I guess that 0xff is a factor similar to the one used in _multiplyAlpha? 

if so, then in this case it's not necessary. by using multiplication and bit shifting u are using approximation and the final 'gray value' is closer to `0.2125*R + 0.7154*G + 0.0721*B` without this factor (in 16% cases we get a value that is greater/smaller by 1, with the `0xff` factor in 92% of all cases the value is a bit different).

also I would add spaces here and in l.58

Reply:
@mmaciola could you please confirm this? (resolve it if it's done)

Reply:
Removed 0xff factor.
I attach the table with comparison.
<img width="947" alt="Luma calc compare" src="https://user-images.githubusercontent.com/71131832/147733438-e8f09c15-e1cf-4b62-bc9f-f01c37f2968a.png">